src/implementations/twig/components/select/select.html.twig

```diff
@@ -0,0 +1,201 @@
+{% set _optional_text = optional_text|default('') %}
+{% set _label = label|default('') %}
+{% set _multiple = multiple|default(false) %}
+{% set _multiple_placeholder = multiple_placeholder|default('') %}
+{% set _multiple_all_text = multiple_all_text|default('') %}
+{% set _multiple_search_text = multiple_search_text|default('') %}
+{% set _compliance = _compliance_|default(false) %}
+
+{% set _css_group_class = 'ecl-form-group' %}
+{% set _css_class = ['ecl-select'] %}
+
+{% if _invalid %}
+  {% set _css_class = _css_class|merge(['ecl-select--invalid']) %}
+{% endif %}
+{% set _css_class = _css_class|join(' ') %}
+
+{% set _extra_attributes = '' %}
+{% if _multiple %}
+  {% set extra_attributes = [
+    { name: 'multiple' },
+    { name: 'data-ecl-auto-init', value: 'Select' },
+    { name: 'data-ecl-select-multiple'},
+    { name: 'data-ecl-select-default', value: _multiple_placeholder },
+    { name: 'data-ecl-select-search', value: _multiple_search_text },
+    { name: 'data-ecl-select-all', value: _multiple_all_text }
+  ]|merge(extra_attributes|default([])) %}
+{% endif %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_group_classes is defined and extra_group_classes is not empty %}
+  {% set _css_group_class = _css_group_class ~ ' ' ~ extra_group_classes %}
+{% endif %}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% if attr.value is defined %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value|e ~ '"' %}
+    {% else %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name %}
+    {% endif %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+<div class="{{ _css_group_class }}">
+  {% if _label is not empty %}
+    <label
+    {% if _id %}
+      for="{{ _id }}"
+    {% endif %}
+      class="ecl-form-label
+      {{- _invalid ? ' ecl-form-label--invalid' -}}
+      {{- _disabled ? ' ecl-form-label--disabled' }}"
+    >
+      {%- block label _label|raw -%}
+      {%- if _required and _required_text is not empty -%}
+        <span class="ecl-form-label__required">{{ _required_text }}</span>
+      {%- elseif not _required and _optional_text is not empty -%}
+        <span class="ecl-form-label__optional">{{ ' ' ~ _optional_text }}</span>
+      {%- endif -%}
+    </label>
+  {% endif %}
+
+  {% if _helper_text is not empty %}
+    <div class="ecl-help-block{{ _disabled ? ' ecl-help-block--disabled' }}">
+      {%- block helper_text _helper_text|raw -%}
+    </div>
+  {% endif %}
+
+  {% if _invalid and _invalid_text is not empty %}
+    <div class="ecl-feedback-message">
+      {%- block invalid_text _invalid_text|raw -%}
+    </div>
+  {% endif %}
+
+  <div class="ecl-select__container
```

Comment:
it seems that the twig template is still using the deprecated code for invalid select (it is visible as you removed deprecated CSS)
ECL code
![image](https://user-images.githubusercontent.com/25579221/98248011-ab90e800-1f74-11eb-8282-6b8d2b7f58bb.png)

Twig code
![image](https://user-images.githubusercontent.com/25579221/98248035-b2b7f600-1f74-11eb-851c-de2c7eb42c70.png)

We should update the twig template to use latest markup. This could be done in this PR or in another one, as you prefer